src/site/assets/js/usa-search.js

```diff
@@ -1,22 +0,0 @@
-var usasearch_config = { siteHandle:"vets.gov_search" };
-var script = document.createElement("script");
-script.src = "https://search.usa.gov/javascripts/remote.loader.js";
-script.async = true;
-document.getElementsByTagName("head")[0].appendChild(script);
-
-(function() {
```

Comment:
I believe this function should not be removed. It is used to detect incompatible browsers and display a warning message located in `top-nav.html`. You could remove lines 1-5 and rename this script file to more accurately reflect the contents

Reply:
Makes perfect sense. @aubreyarcangel, when you have bandwidth, can you make that update per @bshyong's comment? A name like `incompatible-browser.js` or something like that might really help describe what that script does.

Reply:
That is perfect - thank you!